src/components/reorder.main.jsx

```diff
@@ -0,0 +1,174 @@
+/**
+ * Copyright © 2018 Elastic Path Software Inc. All rights reserved.
+ *
+ * This is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this license. If not, see
+ *
+ *     https://www.gnu.org/licenses/
+ *
+ *
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import ReactRouterPropTypes from 'react-router-prop-types';
+import intl from 'react-intl-universal';
+import { withRouter } from 'react-router';
+import Modal from 'react-responsive-modal';
+import CartLineItem from './cart.lineitem';
+import { cortexFetch } from '../utils/Cortex';
+import { login } from '../utils/AuthService';
+import './reorder.main.less';
+
+const Config = require('Config');
+
+class ReorderMain extends React.Component {
+  static propTypes = {
+    history: ReactRouterPropTypes.history.isRequired,
+    productsData: PropTypes.objectOf(PropTypes.any),
+  };
+
+  static defaultProps = {
+    productsData: {},
+  };
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      openModal: false,
+      errorMassages: {},
```

Comment:
Typo. Should be "messages"
```suggestion
      errorMessages: {},
```